public/app/features/alerting/components/UnifiedAlertingPromotion.test.tsx

```diff
@@ -10,8 +10,8 @@ describe('Unified Alerting promotion', () => {
   });
 
   it('should show by default', () => {
-    const promotion = render(<UnifiedAlertingPromotion />);
-    expect(promotion.queryByLabelText('Alert info')).toBeInTheDocument();
+    render(<UnifiedAlertingPromotion />);
+    expect(screen.queryByText('Try out the Grafana 8 alerting!')).toBeInTheDocument();
```

Comment:
When using `toBeInTheDocument` you should always use `screen.getBy...` rather than `queryBy...`. When you test for non existence of an element (should be avoided) the exact opposite is true.